src/database/schema/ChatSchema.ts

```diff
@@ -337,7 +384,10 @@ class ChatSchema implements BaseDbSchema {
       rawJson,
       status,
       isContinuous: false,
+      attachmentJson,
       type: data?.message?.message_type ?? data?.message?.type,
+      attachmentJson,
+      type: 'regular',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` and `type` properties are duplicated here. This could potentially lead to errors or unexpected behavior.

```diff
-      attachmentJson,
-      type: data?.message?.message_type ?? data?.message?.type,
-      attachmentJson,
-      type: 'regular',
+      attachmentJson: attachmentJson,
+      type: data?.message?.message_type ?? data?.message?.type,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->